src/client/dss.py

```diff
@@ -91,36 +128,45 @@ def parse_dss_info(printer, action, target, file):
             dss_info['bucket'] = target[0:pos]
         else:
             dss_info['bucket'] = target
```

Comment:
Where are you populating dss_info['object']? Can you add dss_info['object'] = target[pos:] here?